tests/documentation/test_files/DocumentationStyleBear/bad_file3.py.test

```diff
@@ -30,9 +30,11 @@ class docstring_if_indented():
             """
             This is `if` indented block function.
             """
+            return None
```

Comment:
Suitable `return` statements are added just to show our new padding system neatly. Earlier this wasn't a problem but now the below happens when `return` is not there... 
eg:
official function padding->0,0
```
         def hello_venus(self):
             """
             This is `if` indented block function.
             """
 
 
 def docstring_inner_function(dummy):
```
So this will be amended to
```
         def hello_venus(self):
             """
             This is `if` indented block function.
             """
def docstring_inner_function(dummy):
```
This is the intended behaviour :wink: . But according to PEP-257 module function should have 2 lines in between. which is violated in amended result. Hence it's added just to show paddings neatly, also to avoid the violation of PEP-257. :wink: